tests/test_icestorm/minimal/test_icestorm_0.tcl

```diff
@@ -17,6 +17,8 @@ verilog_defaults -add -defer
 verilog_defaults -pop
 
 synth_ice40  -top top_module
+
+
 write_blif test_icestorm_0.blif
```

Comment:
Where do these empty lines come from? Could it be a jinja snippet that should use `-%` instead of `%` somewhere?

Reply:
As we move to yosys templates this is outdated/removed